### PR TITLE
Fix usage steps in README

### PR DESCRIPTION
**Pull request by wdhif**
Closes https://github.com/openebs/openebs/issues/487

### CODE REVIEW

**Comment by vharsh** [comment]
:shipit: :+1: 
LGTM

File: README.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ Currently, this docker image will pull notebooks from github repository and will
 ### Pull satyamz/docker-jupyter image
```
Comment (vharsh) [inline README.md]: It should be `openebs/easy-jupyter`

Comment (wdhif) [reply README.md]: My bad, it's done.

**Comment by vharsh** [comment]
Good to merge.

**Comment by prateekpandey14** [comment]
@wdhif Thanks for raising PR.
Please stash all 3 commits to single commit.

**Comment by wdhif** [comment]
@prateekpandey14 Done :)

**Comment by vharsh** [comment]
Looks good to me.

**Review by prateekpandey14 — APPROVED** [review]

**Comment by vharsh** [comment]
Thanks @wdhif :1st_place_medal: are you on twitter?

**Comment by wdhif** [comment]
@vharsh Yes I am, https://twitter.com/wassimdhif

**Pull request merged**